jep/202/README.adoc

```diff
@@ -119,6 +115,8 @@ The `VirtualFile.run(Callable)` method, that already existed to optimize Workspa
 It was important to implement this in the S3 plugin in order to ensure that Classic or Blue Ocean UI flows relating to artifacts, including simply opening the index page of a build, did not make a large number of network calls.
 The cache system allows a single S3 metadata call (possibly one HTTP request, depending on page size) to retrieve sufficient information for a complete page render of some or all of the artifact tree, without the rendering code (e.g., in Jenkins core) knowing anything about the storage system.
 
+The `apache-httpcomponents-client-4-api` plugin is extended with a `RobustHTTPClient`
```

Comment:
This has been released (thanks @dwnusbaum) but unfortunately ([INFRA-1600](https://issues.jenkins-ci.org/browse/INFRA-1600)) Javadoc is not published in an unpacked format.